src/activities/drawletters/Drawletters.qml

```diff
@@ -0,0 +1,287 @@
+                                duration: 3000
+                                easing.type: Easing.InOutQuad
+                            }
+                            NumberAnimation {
+                                target: pointImage
+                                property: "rotation"
+                                from: 150; to: -150
+                                duration: 3000
+                                easing.type: Easing.InOutQuad
+                            }
+                        }
+                        SequentialAnimation {
+                            NumberAnimation {
+                                target: pointImage
+                                property: "scale"
+                                from: 1; to: 1.5
+                                duration: 1500
+                                easing.type: Easing.InOutQuad
+                            }
+                            NumberAnimation {
+                                target: pointImage
+                                property: "scale"
+                                from: 1.5; to: 1
+                                duration: 1500
+                                easing.type: Easing.InOutQuad
+                            }
+                            NumberAnimation {
+                                target: pointImage
+                                property: "scale"
+                                from: 1; to: 1.5
+                                duration: 1500
+                                easing.type: Easing.InOutQuad
+                            }
+                            NumberAnimation {
+                                target: pointImage
+                                property: "scale"
+                                from: 1.5; to: 1
+                                duration: 1500
+                                easing.type: Easing.InOutQuad
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+
+        MultiPointTouchArea {
+            anchors.fill: parent
+            minimumTouchPoints: 1
+            maximumTouchPoints: 1
+            z: 100
+
+            function checkPoints(touchPoints) {
+                for(var i in touchPoints) {
+                    var touch = touchPoints[i]
+                    for(var p = 0; p < pointImageRepeater.count; p++) {
+                        var part = pointImageRepeater.itemAt(p)
+
+                        if(touch.x > part.x && touch.x < part.x + part.width &&
+                           touch.y > part.y && touch.y < part.y + part.height) {
+                            part.drawSegment()
+
+                        }
+                    }
+                }
+
+            }
+
+            onPressed: {
+                checkPoints(touchPoints)
+                // to play upon clicking
+                activity.audioEffects.play(Activity.url + "buttonclick."+ApplicationInfo.CompressedAudio)
```

Comment:
you can use "buttonclick.$CA" instead of "buttonclick."+ApplicationInfo.CompressedAudio